modules/radsBidAdapter.js

```diff
@@ -31,33 +30,35 @@ export const spec = {
       let endpoint = isDev ? ENDPOINT_URL_DEV : ENDPOINT_URL;
 
       let payload = {};
-      if (isVideoRequest(bidRequest)) {
-        let vastFormat = params.vastFormat || DEFAULT_VAST_FORMAT;
+      if (isBannerRequest(bidRequest)) {
+        let size = getBannerSizes(bidRequest)[0];
         payload = {
-          rt: vastFormat,
+          rt: 'bid-response',
           _f: 'prebid_js',
           _ps: placementId,
-          srw: width,
-          srh: height,
+          srw: size.width,
+          srh: size.height,
           idt: 100,
           rnd: rnd,
           p: referrer,
           bid_id: bidId,
         };
       } else {
+        let vastFormat = params.vastFormat || DEFAULT_VAST_FORMAT;
```

Comment:
It would be helpful to add vastFormat and other params you accept (ip, pfilter, bcat, dvt, latitude, longitude) to your markdown file and the docs PR.